buildlib/pr/io_demo/io-demo.yml

```diff
@@ -133,61 +39,77 @@ jobs:
-            initial_delay: ${{ coalesce(test.Value.initial_delay, parameters.initial_delay) }}
-            test_interference: ${{ test.Value.interference }}
-            test_with_ece: ${{ coalesce(test.Value.ece, parameters.ece) }}
-      maxParallel: 1
+  - template: jobs-io-demo.yaml
+    parameters:
+      demands: "ucx_iodemo -equals yes"
+      name: "with_interference"
+      tests:
+        "tag match on CX4":
+          duration: 600
+          interface: $(roce_iface_cx4)
+          tls: "rc_x"
+          test_name: tag_cx4_rc
+          extra_run_args: ""
+        "tag match on CX6/RC":
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "rc_x"
+          test_name: tag_cx6_rc
+          extra_run_args: ""
+        "active messages on CX6/RC":
+          args: "-A"
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "rc_x"
+          test_name: am_cx6_rc
+          extra_run_args: ""
+        "tag match on CX6/DC":
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "dc_x"
+          test_name: tag_cx6_dc
+          extra_run_args: ""
+        "active messages on CX6/DC with data validation":
+          args: "-A -q"
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "dc_x"
+          test_name: am_validate_cx6_dc
+          extra_run_args: ""
 
-    variables:
-      workspace: drop_$(Build.BuildId)
-      io_demo_exe: drop_$(Build.BuildId)/install/bin/io_demo
-      cycles: ${{ parameters.cycles }}
-      downtime: ${{ parameters.downtime }}
-      uptime: ${{ parameters.uptime }}
-
-    displayName: "Test "
-    steps:
-      - checkout: none
-        clean: true
-      - task: DownloadBuildArtifacts@0
-        displayName: 'Download Build Artifacts'
-        inputs:
-          artifactName: drop_$(Build.BuildId)
-          downloadPath: $(System.DefaultWorkingDirectory)
-      - bash: chmod u+rwx $(workspace) -R
-      - template: az-stage-io-demo.yaml
-        parameters:
-          name: $(test_name)
-          iodemo_args: $(test_args)
-          duration: $(test_time)
-          roce_iface: $(test_intefrafe)
-          iodemo_tls: $(test_ucx_tls)
-          initial_delay: $(initial_delay)
-          interference: $(test_interference)
-          ece: $(test_with_ece)
-          extra_run_args: $(test_extra_run_args)
-          ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
-            analyzer_allow_list_args: '--allow_list $(System.PullRequest.TargetBranch)'
-      - bash: |
-          rm -rf $(workspace)
-        displayName: Remove artifacts
-        condition: succeeded()
+  - template: jobs-io-demo.yaml
+    parameters:
+      demands: "ucx_iodemo_without_interference -equals yes"
+      name: "without_interference"
+      tests:
+        "tag match on CX4 with user memh":
+          args: "-z"
+          interference: 'No'
+          duration: 120
```

Comment:
can we set "duration" and "interference" parametes once for all "without_interference" tests?
same for "with_interference"

Reply:
do via parameter , looks good